packages/base-types/src/text/index.ts

```diff
@@ -59,3 +59,11 @@ export type AnyElement = Element | LinkElement | VariableElement;
 export type Descendant = AnyElement | Text;
 
 export type SlateTextValue = Descendant[];
+
+export const isVariableElement = (element: Element): element is VariableElement => {
+  return element.type === ElementType.VARIABLE;
+};
+
+export const isLinkElement = (element: Element): element is VariableElement => {
```

Comment:
`element is VariableElement` is this correct? function is called `isLinkElement`

Reply:
Nice catch thank you!